apps/src/sites/hourofcode.com/pages/views/hoc_events_map.js

```diff
@@ -100,35 +100,33 @@ $(document).ready(function() {
     });
 
     map.on('click', 'hoc-events', function(e) {
-      var coordinates = e.features[0].geometry.coordinates.slice();
-      var organizationName = e.features[0].properties.organization_name;
-      var city = e.features[0].properties.city;
+      const coordinates = e.features[0].geometry.coordinates.slice();
+      const organizationName = e.features[0].properties.organization_name;
+      const city = e.features[0].properties.city;
 
       if (popup) {
         popup.remove();
       }
-      const popupText =
-        organizationName + (city.length > 0 ? ' (' + city + ')' : '');
+      const citySuffix = city && city.length > 0 ? ' (' + city + ')' : '';
```

Comment:
Real change here, and in identical code below.

Reply:
Would it make sense to factor out some of this common code to a helper function?

Reply:
It would :), just wanted to get something done quickly. Let me take a quick stab though while I'm here